crates/crabe/src/main.rs

```diff
@@ -145,7 +147,14 @@ impl System {
             self.guard_component
                 .step(&mut self.world, &mut command_map, &mut ToolCommands);
             feedback = self.output_component.step(command_map, ToolCommands);
-            thread::sleep(_refresh_rate);
+            // info!("Execution time : {} μs", &timer.elapsed().as_micros());
+            let elapsed = timer.elapsed();
+            let mut sleep_time = _refresh_rate;
+            if elapsed <= _refresh_rate {
```

Comment:
Il semble y avoir un problème si le temps `elasped` est supérieur à `refresh_rate`, la variable `sleep_time` sera égale au `refresh_rate`. 
Il y aura un sleep qui n'aurait pas du avoir lieu.

Reply:
Aussi vu qu'on utilse maintenant refresh_rate maintenant supprime le todo du fichier.

Reply:
Dans une précédente version, nous avions ça : 
```rust
let elapsed = start.elapsed().as_micros() as f64 / 1000.0;
        let sleep_time =
            Duration::from_millis(16).as_micros() as i128 - start.elapsed().as_micros() as i128;
        if sleep_time > 0 {
            sleep(Duration::from_micros(sleep_time as u64));
        }
```

Reply:
Alright I'll let you commit that on this branch, and I'll cherry-pick it onto #75 if everyone agrees with this. I don't have time at the moment to test it out.